test/integration/dashboard_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class DashboardTest < ActionDispatch::IntegrationTest
-  def setup
+  before(:all) do
```

Comment:
It's intentional, it's not necessary to create a new host & reset `users(:admin).widgets` to default in every integration test in this file. This should make it run only once so that we save some time.